libraries/library/src/main/java/com/chesire/nekome/library/SeriesDomain.kt

```diff
@@ -0,0 +1,31 @@
+package com.chesire.nekome.library
+
+import android.os.Parcelable
+import com.chesire.nekome.core.flags.SeriesStatus
+import com.chesire.nekome.core.flags.SeriesType
+import com.chesire.nekome.core.flags.Subtype
+import com.chesire.nekome.core.flags.UserSeriesStatus
+import com.chesire.nekome.core.models.ImageModel
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * Domain object for a single Series item.
+ */
+@Parcelize
+data class SeriesDomain(
+    val id: Int,
+    val userId: Int,
+    val type: SeriesType,
+    val subtype: Subtype,
+    val slug: String,
+    val canonicalTitle: String,
+    val seriesStatus: SeriesStatus,
+    val userSeriesStatus: UserSeriesStatus,
+    val progress: Int,
+    val totalLength: Int,
+    val posterImage: ImageModel,
+    val startDate: String,
+    val endDate: String
+) : Parcelable {
+    val lengthKnown = totalLength != 0
```

Comment:
Add test to verify this property

Reply:
Actually, maybe this could be an extension method in the Series module, keeps this domain object cleaner